relay-server/src/services/store.rs

```diff
@@ -1189,13 +1238,13 @@ struct SpanKafkaMessage<'a> {
     is_segment: bool,
 
     #[serde(default, skip_serializing_if = "Option::is_none")]
-    measurements: Option<&'a RawValue>,
+    measurements: Option<BTreeMap<&'a str, Option<SpanMeasurement>>>,
```

Comment:
Measurement keys _should_ be able to parse as borrowed, but I would not risk it:

```suggestion
    measurements: Option<BTreeMap<&'a RawValue, Option<SpanMeasurement>>>,
```